### PR TITLE
[13.0] [MIG] partner_contact_nationality

### PR DESCRIPTION
**Pull request by misern2**
Standard migration

### CODE REVIEW

File: partner_contact_nationality/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2014-2015  Grupo ESOC <www.grupoesoc.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Contact nationality",
+    "summary": "Add nationality field to contacts",
+    "version": "13.0.1.0.0",
+    "category": "Customer Relationship Management",
+    "website": "https://odoo-community.org/",
```
Comment (nikul-serpentcs) [inline partner_contact_nationality/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/partner-contact",
```

Comment (misern2) [reply partner_contact_nationality/__manifest__.py]: @nikul-serpentcs Change updated. Thanks!

**Review by HaraldPanten — APPROVED** [review]
LGTM functional review 👍

File: partner_contact_nationality/README.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+the partner contact form.
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+* https://github.com/OCA/partner-contact/
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/10.0
```
Comment (skeller1) [inline partner_contact_nationality/README.rst]: Please regenerate the readme for Odoo 13

File: partner_contact_nationality/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2014-2015  Grupo ESOC <www.grupoesoc.es>
```
Comment (skeller1) [inline partner_contact_nationality/models/__init__.py]: No copyright in init files, maybe you could remove it

**Review by skeller1 — CHANGES_REQUESTED** [review]
Functional review LGTM :+1: 

small changes:
- Regenerate readme for Odoo V13 (it is for V10)
- Remove copyright from init files

Feel free to change the change the license links to secure http protocol (http://www.gnu.org/licenses/agpl.html => https://www.gnu.org/licenses/agpl.html)

Thank you for supporting OCA and your migration

**Comment by misern2** [comment]
@skeller1 All issues are solved!. Thanks.

**Comment by misern2** [comment]
@pedrobaeza Travis is red because of downloading geonames file failure. Any clue how to fix that ?

Thanks

**Comment by pedrobaeza** [comment]
We have to wait a bit until Geonames solves it. It happens once every 2 months or so, and they need one day for solving the problem, but it serves for knowing that using right now `base_location_geonames_import`, it won't work.

**Comment by pedrobaeza** [comment]
Geonames glitch solved

**Review by skeller1 — APPROVED** [review]
Thank you

File: partner_contact_nationality/models/res_partner.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2014-2015  Grupo ESOC <www.grupoesoc.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    nationality_id = fields.Many2one("res.country", "Nationality")
```
Comment (NL66278) [inline partner_contact_nationality/models/res_partner.py:10]: Strictly speaking this is not correct. There are nationalities spread over multiple states, and states containing multiple nationalities. Not blocking.

**Pull request merged**